ddp-workspace/projects/ddp-dsm-ui/src/app/field-datepicker/field-datepicker.component.html

```diff
@@ -29,6 +29,8 @@
           *ngIf="showTodayButton"
           (click)="setToday()"
           [disabled]="disabled">Today</button>
+  <button *ngIf="showSaveBtn" class="btn btn-xs Button--Background--Color ng-star-inserted"
```

Comment:
How do you think, would it not be nicer, to display the button in a different color, which will depend on response status, like so:
If successful - green, if failed - red, if waiting - grey (and disabled)

Reply:
I think that would be too much and we are not doing that anywhere else!
But I agree with while it is saving. we should disable the button till it is done